mirascope/chroma/vectorstores.py

```diff
@@ -0,0 +1,64 @@
+from typing import Any, ClassVar, Optional, Union
+
+from chromadb import Client, Collection
+from chromadb import Settings as ClientSettings
+from chromadb.api import ClientAPI
+
+from mirascope.base.types import Document
+from mirascope.base.vectorstores import BaseVectorStore
+from mirascope.chroma.types import ChromaParams, ChromaQueryResult
+
+
+class ChromaVectorStore(BaseVectorStore):
+    """A vectorstore for Chroma."""
+
+    vectorstore_params = ChromaParams(get_or_create=True)
+    client_settings: ClassVar[ClientSettings] = ClientSettings()
+
+    @property
+    def _client(self) -> ClientAPI:
+        return Client(self.client_settings)
+
+    @property
+    def _index(self) -> Collection:
```

Comment:
please organize private attributes/fields/properties/methods in a private section. See other implementations e.g. `BaseExtractor` for reference

Reply:
do private properties get a PRIVATE PROPERTY section, or throw it all in PRIVATE METHODS

Reply:
Hmm, thinking through this more it doesn't make sense for `_client` and `_index` to be properties at all? You're going to create a client / collection every time?

My initial instinct is to used `cached_property` instead so that it get's cached on the first call. I was hoping maybe we could use `PrivateAttr` or something, but the `default_factory` can't access `cls` or `self`...

And yeah, I would probably have two separate sections, `PRIVATE PROPERTIES` above existing `PRIVATE METHODS`